### ISSUE TITLE
This page has markup errors, causing screen readers to miss content.

### ISSUE THREAD

**Issue by lsgnaija** [workchainio/bug-bounty#9]
Fix the errors listed on the Standards tab of this report. Markup errors like missing end tags mean screen readers may skip important content.
Line 132 WCAG 2.0 A F70 Section 508 (2017) A F70
<img height="1" width="1" style="display:none"

133
src="https://www.facebook.com/tr?id=199903280639076&ev=PageView&noscript=1"

134
/>
